specification/monitor/resource-manager/readme.md

```diff
@@ -121,6 +121,16 @@ go:
 ``` yaml $(go) && $(multiapi)
 batch:
   - tag: package-2017-08
+  - tag: package-2017-09
+```
+
+### Tag: package-2017-09 and go
+
+These settings apply only when `--tag=package-2017-09 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2017-09' && $(go)
+output-folder: $(go-sdk-folder)/services/monitor/mgmt/2018-03-01/insights

```

Comment:
The API version here doesn't match the tags.

Reply:
Ugh, this one sucks. If you take a look through the API Versions they're referencing, there doesn't seem to be any correlation to the tag IDs they chose. I just evaluated `max()` on them and went with that. The big open question here in my mind is what the REST documentation does to handle this, and whether or not it is sufficiently convenient for us to expose their pattern to our customers.

Reply:
Kk, so I just looked it up. You can see the REST documentation here:
https://docs.microsoft.com/en-us/rest/api/monitor/metrics/list

It seems like they make no attempt to view API Versions as wholly contained. For now, I believe the course of action I chose of just going under the moniker of the largest available API Version makes most sense.

Reply:
In cases like this I think it makes sense to align the "API version" with the package version.  Unfortunately we didn't do that for the preview release (it's 2017-05-01-preview).  Using the max API version in a grouping might not give us the desired result.  E.g. imagine a new config, package-2017-10, that's a superset of package-2017-09 adding an API version earlier than 2018-03-01.

Reply:
@jhendrixMSFT and I chatted offline, and agreed to go with the `max` strategy for now. Partially because we'd already released that way, and partially because it'll help keep us as in-sync with documentation as possible. However, there are troubles with doing it this way, because folks can end up slightly out-of-sync, and release a new API Version with an older date associated with it. We'll cross that bridge when we come to it.